app/domain/entities/transfer/transfer.go

```diff
@@ -6,18 +6,20 @@ import (
 )
 
 type Transfer struct {
-	//ID único da transfêrencia
-	ID types.TransferID `json:"id"`
+	//ID único da transfêrencia (Númerico gerado pelo banco)
+	ID int
+
+	External_ID types.TransferID
 
 	//Código da conta de origem
-	Account_origin_id types.AccountOriginID `json:"acount_origin_id"`
+	Account_origin_id types.AccountOriginID
```

Comment:
Há uma orientacao de boas práticas pra nomear variaveis, interfaces, etc, em [Go](https://go.dev/doc/effective_go#names). Aí voce mantem o json pra formatacao da saída, como estava anteriormente.

```
MixedCaps
Finally, the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.
```

Reply:
Esse eu pretendo fazer em outro commit Xenia, pq deixei passar no ajuste do contas.

Reply:
acabei fazendo kkk
Tá lá